koku/api/settings/utils.py

```diff
@@ -274,7 +275,9 @@ def set_currency(schema, currency_code=KOKU_DEFAULT_CURRENCY):
             raise ValueError(f"{currency_code} is not a supported currency")
 
         if not account_currency_setting:
-            set_default_user_settings()
+            overwrite_default = deepcopy(DEFAULT_USER_SETTINGS)
```

Comment:
I probably would have just called this `settings` instead of `overwrite_default`. But either way, these changes look good.